flags-sdk/hypertune/components/dev-tools.tsx

```diff
@@ -0,0 +1,28 @@
+'use client'
+
+import { ArrowPathIcon } from '@heroicons/react/24/outline'
+import { useRouter } from 'next/navigation'
+
+export function DevTools() {
+  const router = useRouter()
+
+  const resetCookie = () => {
```

Comment:
I would simply delete the cookie, so the next request relies on a fresh state? This way we can also safely test that the new cookie gets set properly on first page view with a blank slate.

Reply:
This example uses a server fingerprint and not a cookie, so nonce is undefined by default and only set by devtools to change the hash. In production, this behaviour would be undefined and doesn't need to be tested